frontend/package.json

```diff
@@ -82,6 +82,7 @@
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
     "husky": "4.2.3",
+    "jest-styled-components": "7.0.2",
```

Comment:
Fixes some issues with Jest styled component generated classnames and snapshots.